packages/button/src/css/index.ts

```diff
@@ -74,79 +71,84 @@ export default {
     height: '40px'
   },
   [`.psds-button--size-${vars.sizes.large}`]: {
     fontSize: type.fontSize400,
-    padding: `${layout.spacingXXSmall} ${layout.spacingMedium}`,
+    letterSpacing: type.letterSpacingNone,
+    padding: `${layout.spacingXXSmall} ${layout.spacingLarge}`,
     height: '48px'
   },
   [`.psds-button--appearance-${vars.appearances.primary}.psds-theme--${themeDefaultName}`]: {
     color: colorsTextIcon.highOnDark,
     background: colorsPrimaryAction.background,
     '&:not([disabled]):hover': {
-      background: colorsBlue[7],
+      background: colorsBlue[8],
       cursor: 'pointer'
     },
     '&:not([disabled]):focus': {
-      background: colorsBlue[7],
+      background: colorsBlue[8],
       cursor: 'pointer'
     },
     '&:not([disabled]):active': {
-      background: colorsBlue[8]
+      background: colorsBlue[9]
     }
   },
   [`.psds-button--appearance-${vars.appearances.primary}.psds-theme--${themeNames.light}`]: {
     color: colorsTextIcon.highOnDark,
     background: colorsPrimaryAction.background,
     '&:not([disabled]):hover': {
-      background: colorsBlue[7],
+      background: colorsBlue[8],
       cursor: 'pointer'
     },
     '&:not([disabled]):focus': {
-      background: colorsBlue[7],
+      background: colorsBlue[8],
       cursor: 'pointer'
     },
     '&:not([disabled]):active': {
-      background: colorsBlue[8]
+      background: colorsBlue[9]
     }
   },
   [`.psds-button--appearance-${vars.appearances.secondary}.psds-theme--${themeDefaultName}`]: {
     color: colorsTextIcon.highOnDark,
-    background: colorsSecondaryAction.backgroundDefaultOnDark,
+    background: colorsBackgroundUtility[25],
 
     '&:not([disabled]):hover': {
-      background: colorsSecondaryAction.backgroundHoverOnDark
+      background: colorsBackgroundUtility[30]
     },
     '&:not([disabled]):focus': {
-      background: colorsSecondaryAction.backgroundHoverOnDark
+      background: colorsBackgroundUtility[30]
     },
     '&:not([disabled]):active': {
-      background: colorsSecondaryAction.backgroundPressedOnDark
+      background: colorsBackgroundUtility[20]
     }
   },
   [`.psds-button--appearance-${vars.appearances.secondary}.psds-theme--${themeNames.light}`]: {
     color: colorsTextIcon.highOnLight,
-    background: colorsSecondaryAction.backgroundDefaultOnLight,
+    background: colorsBackgroundUtility[20],
+
     '&:not([disabled]):hover': {
-      background: colorsSecondaryAction.backgroundHoverOnLight
+      background: colorsBackgroundUtility[25]
     },
     '&:not([disabled]):focus': {
-      background: colorsSecondaryAction.backgroundHoverOnLight
+      background: colorsBackgroundUtility[25]
     },
     '&:not([disabled]):active': {
-      background: colorsSecondaryAction.backgroundPressedOnLight
+      background: colorsBackgroundUtility[30]
     }
   },
   [`.psds-button--appearance-${vars.appearances.stroke}.psds-theme--${themeDefaultName}`]: {
```

Comment:
Instead of updating stroke appearance, when do we get to just remove it as once planned?